CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://github.com/AldaronLau/semver).
 
+## [1.2.4] - 2022-11-28
+### Added
```

Comment:
Version will be 1.3.0 to be semver compatible.  I will probably do some extra testing before release so the date can be set to "Unreleased".

Reply:
Done!